data-raw/instream/clear/clear_creek.rmd

```diff
@@ -140,15 +135,13 @@ The following plot shows the weighted usable spawning area in square feet per th
 feet <- cvpiaHabitat::watershed_lengths %>% 
   filter(watershed == 'Clear Creek', lifestage == 'spawning') %>% 
   pull(feet)
-
 names(feet) <- c('FR_spawn_wua', 'SR_spawn_wua', 'ST_spawn_wua')
-
 clear_creek_instream %>%
   select(flow_cfs, SR_spawn_wua) %>%
```

Comment:
Something is off about the units for spring run and steelhead spawning (way too low), we can look at this together when you get back

Reply:
The data in the IFIMWUA sheet and the source document have low values for spawning. I added a note to ask Mark Gard why these values are so low.